_episodes/52.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: episode
+title: "52: Commander Bun Bun takes a bite out of Yugabyte"
+date: 2023-10-26
+tags: trino yugabyte cloud database
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "Episode 52: Commander Bun Bun takes a bite out of Yugabyte"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Denis Magda](https://www.linkedin.com/in/dmagda/), Director of Developer
+  Relations at [Yugabyte](https://www.yugabyte.com/)
+* Anyone else?
+
+## Releases 428-430
+
+Unofficial highlights from Cole:
+
+[Trino 428](https://trino.io/docs/current/release/release-428.html)
+
+* Reduced memory usage for `GROUP BY`
+* Simplified configuration for managing writer counts
+* Faster reads for small Parquet files on data lakes
+* Support for [query options](https://docs.pinot.apache.org/users/user-guide-query/query-options)
+  on dynamic tables in Pinot
+
+[Trino 429](https://trino.io/docs/current/release/release-429.html)
+
+* Faster reading of ORC files in Hive
+* More types supported for schema evolution in Hive
+* Security improvements, including logging out of a session with the Web UI
+
+[Trino 430](https://trino.io/docs/current/release/release-430.html)
+
+* Improved performance of `GROUP BY`
+* Support for setting a timezone on the session level
+* Table statistics in MariaDB
```

Comment:
Do we want to spill the beans on 431 - I think yes. It will have SQL routines and probably come today or tomorrow latest.